packages/webpack/examples/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "app",
```

Comment:
I'm hesitating - to keep the example app or remove it and document using the test fixture a one. 
It's easier to mess around in this app, but it's so much stuff.

Maybe I can clean it up a bit and add a malicious package example to it.